states.yml

```diff
@@ -9,42 +9,47 @@
   id: "102"
   abbr: WBR
 
-- name: Hainaut
+- name: Brussels Capitat Region
```

Comment:
:warning: Is `Capitat` a typo?

Reply:
I copied it from Theo, but yes it might be...